x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/flyout_components/header/translations.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const CREATE_RULE_EXCEPTION = i18n.translate(
+  'xpack.securitySolution.ruleExceptions.flyoutComponents.footer.addException',
```

Comment:
The translations in this file should all be using `header` instead of `footer`, right?

It's more than bit concerning that our i18n tool allows this PR to introduce multiple translations with the same key. Maybe those errors are simply being masked by the current ["orphaned translation" errors](https://buildkite.com/elastic/kibana-pull-request/builds/223300#0190e09d-131f-43b2-accc-82ef2c2577c4), but if those get fixed and this is still allowed, we should probably file an bug with @kbn/i18n.

Reply:
I addressed the other feedback and fixed the initial i18n errors to see if these get reported. I can follow up with a bug if they don't and will update these after that.